src/latexify/frontend.py

```diff
@@ -173,7 +190,7 @@ def expression(
     This function is a shortcut for `latexify.function` with the default parameter
     `use_signature=False`.
     """
-    kwargs["use_signature"] = kwargs.get("use_signature", False)
+    kwargs["style"] = Style.EXPRESSION
```

Comment:
This function no longer need to invoke `function`, just invoking `get_latex` directly is fine.

Reply:
`expression` returns a `LatexifiedFunction`, while `get_latex` returns `str`. I think we still need the extra work that `function` provides to wrap the result of `get_latex`, no?

Reply:
Ah sorry I was a bit confused! I will consider these implementations later.